packages/lesswrong/integrationTests/utils.ts

```diff
@@ -237,7 +237,7 @@ export const createDummyComment = async (user: any, data?: any) => {
     },
   }
   if (!data.postId) {
-    const randomPost = await Posts.findOne()
+    const randomPost = await Posts.findOneArbitrary()
```

Comment:
I actually find `findOne()` easier to understand than `findOneArbitrary()`, because in most cases `findOneArbitrary()` is being used to get a random document, so one could think that it's doing some genuine random sorting under the hood. Whereas `findOne()` makes it clear that it's just doing a query with no selector

Reply:
(this is not introduced in this PR, but I would prefer getting rid of `findOneArbitrary()` altogether)

Reply:
IIRC the reason for making this not part of `findOne` is that if you allow `findOne()` with no arguments, you're also allowing `findOne(selectorFromAVariable)` where you thought there would be a selector but it's nullish.